### PR TITLE
Add Plaid Info product

### PR DESCRIPTION
**Pull request by jollysahil**

### CODE REVIEW

File: spec/plaid_spec.rb [hunk 1]
```diff
@@ -122,4 +122,14 @@
       end
     end
   end
+
+  describe Plaid::Info do
```
Comment (jollysahil) [inline spec/plaid_spec.rb:126]: @justinworth do you happen to know the mystery behind `keys.yml`
I see it referenced in a couple of places, including here - https://github.com/Gusto/plaid-ruby/blob/master/spec/connect_spec.rb#L87
but don't see it in the repo

Comment (justinworth) [inline spec/plaid_spec.rb:126]: i have it on my machine, not sure how i got it, but i got it sometime in december. looks like it's meant to supply data to the plaid_ruby config (normally it comes from the consuming app)

contents:
client_id: test_id
secret: test_secret
base_url: https://tartan.plaid.com/

i dont think testing/ci was ever fully set up for this gem...

**Comment by justinworth** [comment]
@zp-alfred accept

**Comment by zp-alfred** [comment]
Accepted by justinworth!

**Pull request closed without merging**